tuf/api/metadata.py

```diff
@@ -726,6 +726,10 @@ def __init__(
         super().__init__(version, spec_version, expires, unrecognized_fields)
         self.consistent_snapshot = consistent_snapshot
         self.keys = keys
+        mandatory_roles = ["root", "timestamp", "snapshot", "timestamp"]
```

Comment:
Did you mean?
```suggestion
        mandatory_roles = ["root", "timestamp", "snapshot", "targets"]
```
IMO, we should define something like a `TOP_LEVEL_ROLE_NAMES` global for this list somewhere.